omni-relayer/src/defaults.rs

```diff
@@ -0,0 +1,16 @@
+pub const SIGN_TRANSFER_GAS: u64 = 300_000_000_000_000;
+pub const SIGN_TRANSFER_ATTACHED_DEPOSIT: u128 = 500_000_000_000_000_000_000_000;
+
+/// Mainnet
+pub const ETH_RPC_MAINNET: &str = "https://eth.llamarpc.com";
+pub const ETH_WS_MAINNET: &str = "wss://eth-mainnet.g.alchemy.com/v2/API-KEY";
+pub const BRIDGE_TOKEN_FACTORY_ADDRESS_MAINNET: &str = "0x252e87862A3A720287E7fd527cE6e8d0738427A2";
+
+/// Testnet
+pub const NEAR_RPC_TESTNET: &str = "https://rpc.testnet.near.org/";
+pub const ETH_RPC_TESTNET: &str = "https://ethereum-sepolia.blockpi.network/v1/rpc/public";
+pub const ETH_CHAIN_ID_TESTNET: u64 = 11_155_111;
+pub const TOKEN_LOCKER_ID_TESTNET: &str = "omni-locker.test1-dev.testnet";
+pub const BRIDGE_TOKEN_FACTORY_ADDRESS_TESTNET: &str = "0xCF6462b9fcE5aF3E6c660c83453ECa18Ff468773";
+pub const NEAR_LIGHT_CLIENT_ETH_ADDRESS_TESTNET: &str =
+    "0x202cdf10bfa45a3d2190901373edd864f071d707";
```

Comment:
Let's move these consts to config file

Reply:
Did you mean to create a new module like `config.rs` and move these constants there or you wanted to created a real config file like `config.toml`?

Reply:
a full config file like config.toml or config.json

Reply:
Now it feels a bit odd to have both defaults.rs and config.toml. Should I move everything to config.toml?